clustercontroller-core/src/test/java/com/yahoo/vespa/clustercontroller/core/hostinfo/HostInfoTest.java

```diff
@@ -49,11 +52,16 @@ public void testFullSet() throws IOException {
         assertThat(storageNodeList.size(), is(2));
         assertThat(storageNodeList.get(0).getIndex(), is(0));
         List<Metrics.Metric> metrics = hostInfo.getMetrics().getMetrics();
-        assertThat(metrics.size(), is(2));
-        Metrics.Value value = metrics.get(0).getValue();
-        assertThat(value.getLast(), is(5095L));
+        assertThat(metrics.size(), is(4));
+        assertThat(metrics.get(0).getValue().getLast(), is(5095L));
         assertThat(metrics.get(0).getName(), equalTo("vds.datastored.alldisks.buckets"));
+        assertThat(metrics.get(3).getValue().getLast(), is(129L));
+        assertThat(metrics.get(3).getName(), equalTo("vds.datastored.bucket_space.buckets_total"));
         assertThat(hostInfo.getClusterStateVersionOrNull(), is(123));
+
+        Optional<Metrics.Value> value = hostInfo.getMetrics()
+                .getValueAt("vds.datastored.bucket_space.buckets_total", Map.of("bucketSpace", "default"));
+        assertThat(value.map(Metrics.Value::getLast), equalTo(Optional.of(129L)));
```

Comment:
For completeness, consider also adding a check that the `global` `bucketSpace` dimension has zero buckets.

Reply:
Done